Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -370,7 +399,8 @@ import BraintreeCore
             throw BTVenmoError.disabled
         }
 
-        if !isVenmoAppInstalled() {
+
+        if !shouldFallback && !isVenmoAppInstalled() {
```

Comment:
Nit - it's a little cleaner IMO to pass the `shouldFallback` bool as a param to `verifyAppSwitch()` versus storing it as a property

Reply:
That's a great callout, updated: https://github.com/braintree/braintree_ios/pull/1193/commits/d02955e587213989e0d059df2dd3834a461bbe6b